src/main/java/com/band/api/resolvers/Mutation.java

```diff
@@ -1,21 +1,28 @@
 package com.band.api.resolvers;
 
+import com.band.api.Exceptions.CustomException;
 import com.band.api.domain.User;
 import com.band.api.repository.UserRepository;
 import com.coxautodev.graphql.tools.GraphQLMutationResolver;
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessResourceFailureException;
+import org.springframework.security.crypto.password.PasswordEncoder;
 import org.springframework.stereotype.Component;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 
+@Slf4j
 @Component
 public class Mutation implements GraphQLMutationResolver {
 
 
     private final UserRepository userRepository;
+    private final PasswordEncoder encoder;
 
     @Autowired
-    public Mutation(UserRepository userRepository){
+    public Mutation(UserRepository userRepository, PasswordEncoder encoder){
```

Comment:
Move these dependencies to a UserService class, and make UserService the only dependency in Mutation for now.|

Is there a way to make a UserMutation class and have multiple Mutation classes active at once?